index.html

```diff
@@ -149,7 +151,7 @@ <h4>
 <!--Team-->
 <div class="section scrollspy" id="team">
     <div class="container">
-        <h2>The Impact Dev Team </h2>
+        <h2 class="center">The Impact Dev Team </h2>
```

Comment:
This isn't relevant to this PR, if you think this should change please propose it separately.